readme.md

```diff
@@ -601,6 +601,7 @@
 - [Humane Technology](https://github.com/engagingspaces/awesome-humane-tech#readme) - Open source projects that help improve society.
 - [Speakers](https://github.com/karlhorky/awesome-speakers#readme) - Conference and meetup speakers in the programming and design community.
 - [Software Patreons](https://github.com/uraimo/awesome-software-patreons#readme) - Fund individual programmers or the development of open source projects.
+- [Prisma](https://github.com/catalinmiron/awesome-prisma#readme) - Database tools for modern application development
```

Comment:
The descriptions needs to be more informative and less of a tagline. You also forgot the dot at the end.

Reply:
Done. Please take another look. Thanks!